src/Skeletor/Console/CreateProjectCommand.php

```diff
@@ -42,7 +42,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->setupCommand();
 
         //Start process in the background
-        $process = new Process('php skeletor package:show --no-ansi');
+        $process = new Process('skeletor package:show --no-ansi');
```

Comment:
Yes, this is a separate/new bug. So on composer global, the packages aren't fetched because it cant find the command. When removing the `php ` it works, because it can find the command again.